modules/router/src/admin/index.ts

```diff
@@ -49,6 +49,37 @@ export class AdminHandlers {
       }),
       this.routerAdmin.getMiddlewares.bind(this.routerAdmin),
     );
+    this.routingManager.route(
+      {
+        path: '/router/route-middlewares',
+        action: ConduitRouteActions.GET,
+        description: `Returns the middleware of an app route.`,
+        queryParams: {
+          path: ConduitString.Required,
```

Comment:
This is REST-oriented.
Not saying we could/should change this, but we should probably document it a bit.

As is, this should include the module name in the path.
Does this work for hooks? What's the path for a hook given the order swap going on.